addons/website/static/src/builder/plugins/form/form_option.xml

```diff
@@ -224,6 +224,104 @@
+            preview="false"
+            dataAttributeAction="'requirementComparator'"
+            action="'setRequirementComparator'"
+        >
+            <BuilderSelectItem id="'no_requirement_opt'" dataAttributeActionValue="''">None</BuilderSelectItem>
+            <BuilderSelectItem dataAttributeActionValue="'substring'">Contains</BuilderSelectItem>
+            <BuilderSelectItem dataAttributeActionValue="'!substring'">Doesn't contain</BuilderSelectItem>
+        </BuilderSelect>
+        <BuilderSelect
+            t-if="domStateCurrentFieldInput.type === 'number'"
+            id="'require_condition_num_opt'"
+            preview="false"
+            dataAttributeAction="'requirementComparator'"
+            action="'setRequirementComparator'"
+        >
+            <!-- number comparator possibilities -->
+            <BuilderSelectItem id="'no_requirement_opt'" dataAttributeActionValue="''">None</BuilderSelectItem>
+            <BuilderSelectItem dataAttributeActionValue="'greater'">Is number greater than</BuilderSelectItem>
+            <BuilderSelectItem dataAttributeActionValue="'less'">Is number less than</BuilderSelectItem>
+            <BuilderSelectItem dataAttributeActionValue="'greater or equal'">Is number greater than or equal to</BuilderSelectItem>
+            <BuilderSelectItem dataAttributeActionValue="'less or equal'">Is number less than or equal to</BuilderSelectItem>
+        </BuilderSelect>
+        <BuilderSelect
+            t-if="domStateCurrentFieldInput.hasDateTimePicker"
+            id="'require_condition_time_comparators_opt'"
+            preview="false"
+            dataAttributeAction="'requirementComparator'"
+            action="'setRequirementComparator'"
+        >
+            <!-- date & datetime comparator possibilities -->
+            <BuilderSelectItem id="'no_requirement_opt'" dataAttributeActionValue="''">None</BuilderSelectItem>
+            <BuilderSelectItem dataAttributeActionValue="'after'">Is date after</BuilderSelectItem>
+            <BuilderSelectItem dataAttributeActionValue="'before'">Is date before</BuilderSelectItem>
+            <BuilderSelectItem dataAttributeActionValue="'equal or after'">Is date after or equal to</BuilderSelectItem>
+            <BuilderSelectItem dataAttributeActionValue="'equal or before'">Is date before or equal to</BuilderSelectItem>
+            <BuilderSelectItem dataAttributeActionValue="'between'">Is date between (included)</BuilderSelectItem>
+            <BuilderSelectItem dataAttributeActionValue="'!between'">Is date not between (excluded)</BuilderSelectItem>
+        </BuilderSelect>
+        <BuilderButton
+            id="'error_message_opt'"
+            title.translate="Add predefined error message"
+            action="'setCustomErrorMessage'"
+            t-if="!isActiveItem('no_requirement_opt')">
+                <i class="fa fa-eye" t-att-class="{'fa-eye-slash': !isActiveItem('error_message_opt')}"/>
```

Comment:
Should not the fa-eye be conditional to the opposite of fa-eye-slash? It may work like that but it should be using one class only